data/templates/css/wikimedia.scss

```diff
@@ -43,6 +43,7 @@ $divider-color-composite: #959595;
 }
```

Comment:
Try to break your commit messages lines, you can find some good guidelines [here](https://github.com/endlessm/eos-documentation/wiki/Commit-Message-Practices)